source/manual/mongodb.html.md

```diff
@@ -5,37 +5,45 @@ layout: manual_layout
 parent: "/manual.html"
 section: Backups
 type: learn
-last_reviewed_on: 2019-08-06
+last_reviewed_on: 2020-03-02
 review_in: 6 months
 ---
 
-We have 2 ways of taking MongoDB backups.
+There are two ways of taking MongoDB backups.
```

Comment:
Aren't there three listed here? 🤔

 * automongodbbackup
 * mongodumps to S3
 * mongodumps via `govuk_env_sync` in AWS?

Reply:
The header formatting makes it a bit ambiguous. "mongodumps via `govuk_env_sync` in AWS" is formatted as a sub-heading of "mongodumps". 

This is likely to distinguish between the two types of mongodumps and the `automongodbbackup` third-party script. 

It boils down to a stylistic choice: yes, there are technically three ways but they fall into two main approaches.

I'm not fussed. 🤷‍♀